Package.swift

```diff
@@ -0,0 +1,46 @@
+// swift-tools-version:5.5
+
+/*
+ Licensed to the Apache Software Foundation (ASF) under one
+ or more contributor license agreements.  See the NOTICE file
+ distributed with this work for additional information
+ regarding copyright ownership.  The ASF licenses this file
+ to you under the Apache License, Version 2.0 (the
+ "License"); you may not use this file except in compliance
+ with the License.  You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing,
+ software distributed under the License is distributed on an
+ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ KIND, either express or implied.  See the License for the
+ specific language governing permissions and limitations
+ under the License.
+ */
+
+import PackageDescription
+
+let package = Package(
+    name: "cordova-plugin-device",
+    platforms: [.iOS(.v11)],
+    products: [
+        .library(
+            name: "cordova-plugin-device",
+            targets: ["cordova-plugin-device"])
+    ],
+    dependencies: [
+        .package(url: "https://github.com/apache/cordova-ios.git", branch: "master")
```

Comment:
This could potentially be an issue since we can't distribute stuff pointing to the development branch, and it's probably bad if this doesn't match the version of CordovaLib being used in the app.

I guess SwiftPM has no concept of anything like "peer dependencies"? 😕

Reply:
I've removed it since by keeping `CordovaLib` in the app project, having the dependency is not needed.

We will miss having plugins that are "standalone" so the source code can be built with xcodebuild without adding them to an app, but that is not possible at the moment neither, so not a big miss.